.github/workflows/ai-runner-models.yaml

```diff
@@ -33,13 +33,28 @@ jobs:
 
       - name: dl_checkpoints.sh
         env:
-          AI_RUNNER_COMFYUI_IMAGE: "livepeer/ai-runner:live-app-comfyui"
+          AI_RUNNER_COMFYUI_IMAGE: "livepeer/ai-runner:live-app-comfyui-89570ca"
```

Comment:
Pinning last working version as the newest changes ([comfystream v0.0.4 ai-runner port](https://github.com/livepeer/ai-runner/pull/479/files)) started causing failures of `dl_checkpoints.sh`